@xen-orchestra/lite/src/views/vm/VmGeneralInfo.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <UiCard class="vm-general-info">
+    <UiTitle>
+      {{ $t('general-information') }}
+    </UiTitle>
+    <VtsQuickInfoRow :label="$t('name')" :value="vm?.name_label" />
+    <VtsQuickInfoRow :label="$t('id')" :value="vm?.uuid" />
+    <VtsQuickInfoRow :label="$t('description')" :value="vm?.name_description" />
+    <VtsQuickInfoRow :label="$t('tags')">
+      <template v-if="vm && vm.tags.length > 0" #value>
+        <UiTagsList>
+          <UiTag v-for="tag in vm.tags" :key="tag" accent="info" variant="secondary">{{ tag }}</UiTag>
+        </UiTagsList>
+      </template>
+    </VtsQuickInfoRow>
+    <VtsQuickInfoRow :label="$t('os-name')" :value="vm?.reference_label" />
+    <VtsQuickInfoRow :label="$t('os-kernel')" :value="guestMetrics?.os_version.uname" />
+    <VtsQuickInfoRow :label="$t('management-agent-version')" :value="PvVersion" />
+  </UiCard>
+</template>
+
+<script setup lang="ts">
+import type { XenApiVm } from '@/libs/xen-api/xen-api.types'
+import { useVmGuestMetricsStore } from '@/stores/xen-api/vm-guest-metrics.store'
+import VtsQuickInfoRow from '@core/components/quick-info-row/VtsQuickInfoRow.vue'
+import UiCard from '@core/components/ui/card/UiCard.vue'
+import UiTag from '@core/components/ui/tag/UiTag.vue'
+import UiTagsList from '@core/components/ui/tag/UiTagsList.vue'
+import UiTitle from '@core/components/ui/title/UiTitle.vue'
+import { computed } from 'vue'
+
+const { vm } = defineProps<{ vm: XenApiVm | undefined }>()
```

Comment:
```suggestion
const { vm } = defineProps<{ vm: XenApiVm }>()
```

I do not think it's necessary since we always have a vm id when we click on it